### ISSUE TITLE
Piece movement issue after openings chat on mobile app

### ISSUE THREAD

**Issue by lichessplayer** [veloce/lichobile#1459]
Android Mobile app, when I play a game and open the chat function, either to read a message or send a message, the game there forward lags, premoves don't seem to work, pieces drop onto squares before I have finished dragging my finger to my chosen square.

**To Reproduce**
1. Go to any game
2. Click on the chat box
3. Read or send a message 

**Expected behavior**
The game to continue as normal, premoves working, pieces move smoothly 
**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S9+
 - OS: Android 10
 - App
 - Version 7.5.0

**Additional context**

**Comment by ikazStore** [comment]
This issue always happens to me.

**Comment by LichessUser** [comment]
Yes, I also encounter this issue many times, so what I do is I don't open chat while games even if my opponent sends me a message.

I hope this issue fixes soon.

Thank y'all, love from India🇮🇳

**Comment by cherouvim** [comment]
I consistently reproduce this bug on my OnePlus 8T. Whenever I visit the chat, or go to see the opponent's profile, then returning back to the game heavily affects my ability to make moves. Whether I try to make moves using 2 touches (from, to) or whether I try to drag my pieces, the situation is really bad because I need to try 2 or 3 times until I manage to actually make the move happen.

**Comment by cherouvim** [comment]
This is still an issue. I've lost lots of games due to this but. Note that it doesn't only happen when opening the chat, but whenever switching to another application mid game.
